weather/script.js

```diff
@@ -0,0 +1,119 @@
+let formInput = document.forms.data;
+let cityInput = formInput.elements.city;
+let countryInput = formInput.elements.country;
+let content = document.querySelector('#content');
+let myWeather = document.querySelector('#myWeather-info');
+let blockMyWeather = document.querySelector('#block-myWeather');
+let weatherData = JSON.parse(localStorage.getItem('weather'));
+
+
+for(key in weatherData) {
+    let div = document.createElement('div');
+    div.className = 'myWeather-info';
+    div.innerText =`${key}: temperature ${weatherData[key].temperature}°C, time: ${weatherData[key].observation_time}, wind degree: ${weatherData[key].wind_degree}, humidity: ${weatherData[key].humidity}%, wind_dir: ${weatherData[key].wind_dir}`
+    myWeather.append(div);
+}
```

Comment:
Showing the stored results only available after the page reload, but not before:

![image](https://user-images.githubusercontent.com/31325372/88072808-2842b680-cb7e-11ea-89d6-a7f30c730439.png)

Info is empty even after several search attempts..
Should be updated after each search request

Reply:
This data is hardly readable:

![image](https://user-images.githubusercontent.com/31325372/88072990-5c1ddc00-cb7e-11ea-965c-70620bfb31a7.png)

Please, think about styling this